apps/cowswap-frontend/src/modules/trade/hooks/useBuildTradeDerivedState.ts

```diff
@@ -1,13 +1,13 @@
 import { Atom, useAtomValue } from 'jotai'
 import { useMemo } from 'react'
 
-import { useCurrencyAmountBalance } from '@cowprotocol/balances-and-allowances'
 import { tryParseFractionalAmount } from '@cowprotocol/common-utils'
 import { useTokenBySymbolOrAddress } from '@cowprotocol/tokens'
 import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 
 import { Nullish } from 'types'
 
+import { useCurrencyAmountBalanceCombined } from 'modules/swap/hooks/useCurrencyAmountBalanceCombined'
```

Comment:
`trade` module is global relatively to `swap` and should not depend on it

Reply:
Now this is imported from the new `combinedBalances` module. Is this ok or should we import in a different way?